[10.0] partner_changeset - Fix condition on test_enable

yvaucher opened the pull request:
Context key `test_enable` will always be empty.

Thus previous patch had no effect.
Still this might speed the tests of other modules as we will only trigger changeset methods in thoses tests.

yvaucher committed:
Fix condition on test_enable

yvaucher commented:
See other example here:

https://github.com/OCA/account-invoicing/blob/10.0/sale_timesheet_invoice_description/models/sale.py#L59

pedrobaeza commented:
Why Travis is red?

yvaucher commented:
@pedrobaeza no idea the fixed condition seems right to me now.

By pass change set if:
Specificaly using __no_changeset
or
we are testing a module which is not this one.

Or maybe we are missing some definition of the context `test_partner_changeset`

grindtildeath reviewed (approved):
I approve although travis is red because `self.env.context.get('test_enable')` doesn't make much sense.

yajo reviewed (changes requested):
This PR breaks the branch.

NL66278 commented:
@yvaucher Do you still intend to fix this branch?

yvaucher commented:
@NL66278 thx for the reminder I'll have to dive into that again. It has been a while.

NL66278 commented:
Any update on this? Travis errors are in the tests for this module...

yvaucher committed:
force partner_changeset tests to use test condition

test_enable is there to be nice to other modules.

But we want the tests of partner_changset module to actually test
partner_changeset features.

yvaucher commented:
@NL66278 finally fixed it, it was stupidly missing the use of the context key in the tests.

Should be green now.

:heavy_plus_sign:  rebase

yvaucher commented:
/ocabot merge patch

pedrobaeza commented:
There has been a problem with merge bot, but it's fixed now, so you can relaunch.

yvaucher commented:
/ocabot merge patch
